common.mk

```diff
@@ -3,6 +3,18 @@
 #########################################################################################
 SHELL=/bin/bash
 
+#########################################################################################
+# extra make variables/rules from subprojects
+#
+# EXTRA_GENERATOR_REQS - requirements needed for the main generator
```

Comment:
This is currently unused, I guess this is for dromajo?

Reply:
`GENERATOR_REQS` is for Ariane